packages/gp2-components/src/templates/UserOverview.tsx

```diff
@@ -0,0 +1,85 @@
+import { gp2 } from '@asap-hub/model';
+import {
+  Card,
+  crossQuery,
+  Headline3,
+  pixels,
+  TagList,
+} from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import ExpandableText from '../molecules/ExpandableText';
+import EmailSection from '../organisms/EmailSection';
+
+type UserOverviewProps = Pick<
+  gp2.UserResponse,
+  'email' | 'secondaryEmail' | 'biography' | 'keywords' | 'fundingStreams'
+>;
+
+const { rem } = pixels;
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: rem(32),
+  padding: `${rem(32)} 0 ${rem(48)}`,
+});
+
+const contentStyles = css({
+  paddingTop: rem(32),
+});
+
+const columnStyles = css({
+  display: 'grid',
+  columnGap: rem(32),
+  gridRowGap: rem(32),
+  [crossQuery]: {
+    gridTemplateColumns: '1fr 1fr',
+    rowGap: rem(32),
+  },
+});
+const cardStyles = css({ padding: `${rem(32)} ${rem(24)}` });
+const UserOverview: React.FC<UserOverviewProps> = ({
+  biography,
+  email,
+  secondaryEmail,
+  keywords,
+  fundingStreams,
+}) => (
+  <div css={containerStyles}>
+    <div css={columnStyles}>
+      <Card overrideStyles={cardStyles}>
```

Comment:
to avoid the overrideStyles, should we just pass this to a container div in the card?

Reply:
done